pkg/gendoc/markdown.go

```diff
@@ -60,7 +60,7 @@ __Version:__ {{ .Version }}
 
```

Comment:
also please dont mix generated and handwritten code - the v1.json is in this commit and it should not be.